salt/config/__init__.py

```diff
@@ -371,6 +371,10 @@ def _gather_buffer_space():
     # The number of minutes between mine updates.
     'mine_interval': int,
 
+    'mp_logging_port': int,
+
+    'mp_logging_consumer': bool,
```

Comment:
I was going to ask why you added this, but a sip of coffee later I remembered, test suite logging consumer...